test/lib/move.spec.ts

```diff
@@ -18,7 +18,8 @@ describe('lib/move.ts', () => {
     [{ id: 'chu-id', name: 'c-h-u', lineage: ['from-sub'], type: 'remote' }],
   );
 
-  it('move CHU: success', async () => {
+  it('move CHU: success', async () => {    
+    process.env.ENCRYPTION_KEY = 'superimportantkey';
```

Comment:
you should write your code so modifying global state isn't required for testing

the way i like to do this, is that `main.ts` or whatever your "entrypoint" is should read in all global state and then that data should be injected downward. this makes it so your `main.ts` can't be unit-tested but everything else can